core/domain/classifier_services.py

```diff
@@ -359,62 +416,56 @@ def delete_classifier_training_job(job_id):
         classifier_training_job_model.delete()
 
 
-def get_classifier_from_exploration_attributes(exp_id, exp_version,
-                                               state_name):
-    """Gets the classifier object from the exploration attributes.
+def get_classifier_training_jobs(exp_id, exp_version, state_names):
+    """Gets the classifier training job object from the exploration attributes.
 
     Args:
         exp_id: str. ID of the exploration.
         exp_version: int. The exploration version.
-        state_name: str. The name of the state to which the classifier
-            belongs.
+        state_names: list(str). The state names for which we retrieve the job.
 
     Returns:
-        ClassifierData. Domain object for the Classifier model.
+        list(ClassifierTrainingJob). Domain objects for the Classifier training
+            job model.
 
     Raises:
-        Exception: Entity for class ClassifierExplorationMapping with id not
+        Exception: Entity for class TrainingJobExplorationMapping with id not
             found.
-        Exception: Entity for class ClassifierData with id not found.
+        Exception: Entity for class ClassifierTrainingJob with id not found.
     """
-    classifier_exploration_mapping_model = (
-        classifier_models.ClassifierExplorationMappingModel.get_model(
-            exp_id, exp_version, state_name))
-    classifier_id = classifier_exploration_mapping_model.classifier_id
-    classifier = get_classifier_by_id(classifier_id)
-    return classifier
-
-
-def create_classifier_exploration_mapping(exp_id, exp_version, state_name,
-                                          classifier_id):
-    """Creates an entry for Classifier Exploration Mapping in datastore.
+    training_job_exploration_mapping_models = (
+        classifier_models.TrainingJobExplorationMappingModel.get_models(
+            exp_id, exp_version, state_names))
+    if training_job_exploration_mapping_models is None:
```

Comment:
Added test case for this. @seanlip This depends on the solution the comment above.